playbooks/cluster/openshift/vars/gluster.yml

```diff
@@ -0,0 +1,11 @@
+# Namespace to create for the GlusterFS and heketi resources
```

Comment:
s/heketi/Heketi

Reply:
Because that's how I've seen each stylized. I can keep heketi capitalized for consistency in this repo, at least.